napari/_app_model/actions/_layer_actions.py

```diff
@@ -0,0 +1,164 @@
+"""This module contains actions (functions) that operate on layers.
```

Comment:
Maybe suggest in the comment when should devs modify e.g. `LAYER_ACTIONS`.  

For example, it's not really intended to be modified during runtime. Same thing applies to `constants` etc.

```suggestion
"""This module contains defined actions (functions) that operate on layers.
These are registered by `NapariApplication` when it's first instanced. 
```